snafu/dns_perf_wrapper/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+RUN dnf install -y --nodocs python3 python3-pip epel-release && \
```

Comment:
Can we just switch to a single `dnf install`

Reply:
we need epel-release to install a couple of other dependencies which is why we had to go with multiple dnf commands.